typing/types.ml

```diff
@@ -351,3 +351,12 @@ type label_description =
     lbl_loc: Location.t;
     lbl_attributes: Parsetree.attributes;
    }
+
+let rec bound_value_identifiers = function
```

Comment:
Remember to copy the changes to this function when you rebase over the no-positions-in-paths work.